ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.view.js

```diff
@@ -0,0 +1,152 @@
+                            value: metacardType.id
+                        };
+                    }),
+                    values: this.model.get('descriptors'),
+                    value: [currentValue],
+                    id: 'Attributes'
+                })
+            }));
+        },
+        setupTitleInput: function () {
+            let currentValue = this.model.get('title') ? this.model.get('title') : '';
+            this.basicTitle.show(new PropertyView({
+                model: new Property({
+                    value: [currentValue],
+                    id: 'Title',
+                    placeholder: 'Result Form Title'
+                })
+            }));
+        },
+        setupDescription: function () {
+            let currentValue = this.model.get('description') ? this.model.get('description') : '';
+            this.basicDescription.show(new PropertyView({
+                model: new Property({
+                    value: [currentValue],
+                    id: 'Description',
+                    placeholder: 'Result Form Description'
+                })
+            }));
+        },
+        handleAttributeValue: function () {
+            this.$el.toggleClass('is-type-specific', attribute === 'specific');
+        },
+        turnOnLimitedWidth: function () {
+            this.regionManager.forEach(function (region) {
+                if (region.currentView && region.currentView.turnOnLimitedWidth) {
+                    region.currentView.turnOnLimitedWidth();
+                }
+            });
+        },
+        turnOffEdit: function () {
+            this.regionManager.forEach(function (region) {
+                if (region.currentView && region.currentView.turnOffEditing) {
+                    region.currentView.turnOffEditing();
+                }
+            });
+        },
+        edit: function () {
+            this.$el.addClass('is-editing');
+            this.regionManager.forEach(function (region) {
+                if (region.currentView && region.currentView.turnOnEditing) {
+                    region.currentView.turnOnEditing();
+                }
+            });
+        },
+        cancel: function(){
+            this.cleanup();
+        },
+        save: function () {
+            let view = this;
+            Loading.beginLoading(view);
+            let descriptors = this.basicAttributeSpecific.currentView.model.get('value'); 
+            let title = this.basicTitle.currentView.model.getValue()[0];
+            let description = this.basicDescription.currentView.model.getValue()[0];
+            let templatePerms = {
+                'descriptors': descriptors,
+                'title' : title,
+                'description': description
+            };
+            this.updateResults(templatePerms);
+        },
+        updateResults: function(templatePerms) {
+            let resultEndpoint = `/search/catalog/internal/forms/result/${this.options.modelId}`
```

Comment:
this.options.modelId isn't necessary since that data exists implicitly on the model that is already being passed in:
update: `this.model.id` should be sufficient.

Don't forget to remove any references to modelId where the lightbox is created.